src/Framework/MSBuildEventSource.cs

```diff
@@ -423,6 +423,18 @@ public void ResolveAssemblyReferenceNodeConnectStop()
         {
             WriteEvent(57);
         }
+
+        [Event(58)]
+        internal void ResolveAssemblyReferenceServiceRequestStart()
```

Comment:
Not sure what you mean? Include content of the request?

Reply:
If I remember correctly, ETW events can take an arbitrary object as a parameter, and then it would show up (stringified? not sure) in the trace when you look at it later. It might be helpful if something goes wrong, and we want to see whether the inputs were wrong, RAR did something wrong, or the outputs were used incorrectly afterwards. It does make it slower and larger, though, so it might not be justified.